Std/Tactic/Exact.lean

```diff
@@ -0,0 +1,19 @@
+/-
+Copyright (c) 2023 Mario Carneiro. All rights reserved.
```

Comment:
Is this the right copyright owner + author if you wrote the file?

Reply:
Mario wrote this but asked me to PR it!